.ci/helper_shellchecks.sh

```diff
@@ -5,7 +5,7 @@ CI_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "${CI_DIR}/common.sh"
 
 # shellcheck disable=2016
-mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' && git submodule --quiet foreach '[ "$path" = "base" -o "$path" = "platform/android/luajit-launcher" ] || git grep -lE "^#!(/usr)?/bin/(env )?(bash|sh)" | sed "s|^|$path/|"' && git ls-files ./*.sh; } | sort | uniq)
+mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' && git submodule --quiet foreach '[ "$path" = "base" -o "$path" = "platform/android/luajit-launcher" ] || git grep -lE "^#!(/usr)?/bin/(env )?(bash|sh)" | sed "s|^|$path/|"' && git ls-files ./*.sh; } | sed "/^plugins\/koterm.koplugin\/shfm$/d" | sort | uniq)
```

Comment:
Is there a problem with using the -o mechanism?

Reply:
Maybe I am too stupid, but I could not get it to work.

Reply:
Oh, I forgot to look at this, but in any case:
```suggestion
mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' && git submodule --quiet foreach '[ "$path" = "base" -o "$path" = "platform/android/luajit-launcher" ] || git grep -lE "^#!(/usr)?/bin/(env )?(bash|sh)" | sed "s|^|$path/|"' && git ls-files ./*.sh; } | sed "/^plugins\/terminal.koplugin\/shfm$/d" | sort | uniq)
```

Reply:
Please write the exclusion like this instead:

git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' -- ':(exclude)plugins/terminal.koplugin/shfm'